lib/cass_schema/version.rb

```diff
@@ -1,3 +1,3 @@
 module CassSchema
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
I guess under proper semantic versioning this would be a minor bump. This tries makes it clearer that `_setup` isn't really public.